style/rails/README.md

```diff
@@ -19,8 +19,15 @@ Rails
 * Use `def self.method`, not the `scope :method` DSL.
 * Use the default `render 'partial'` syntax over `render partial: 'partial'`.
 * Use `link_to` for GET requests, and `button_to` for other HTTP verbs.
+* If you break up an argument list for a macro,
+  keep the first argument on the same line as the macro,
```

Comment:
So all the new examples are great, but I'd add another with multiple non-hash options together on one line.